src/ui/SearchInputHeader/__snapshots__/SearchInputHeader.tests.js.snap

```diff
@@ -182,11 +182,12 @@ exports[`The Onboarding LandingScreen component should be stylable 1`] = `
           style={
             Object {
               "flexGrow": 1,
-              "fontFamily": "Colfax-Medium",
+              "fontFamily": "System",
```

Comment:
Again with the font.

Reply:
probably going from custom stuff to core search stuff, we definitely need to take a look

Reply:
Fixed this as well.